uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -103,6 +106,7 @@ private void addAcceptedJobResult(String jobId) {
         JobResult jobResult = new JobResult();
         jobResult.setJobId(jobId);
         jobResult.setStatus(JobStatus.ACCEPTED);
+        jobManager.putJob(jobResult);
```

Comment:
@Rikkola The 2 significant changes I see is this line and removal of `@Inject SpaceAPI`. Could you please explain how that fixes the issue? Also now (provided that you think this should fix the issue) we probably don't need [this workaround](https://github.com/kiegroup/kie-wb-distributions/pull/757) - could you please remove it to make the tests a bit faster? (CC @cristianonicolai just so he knows)

Reply:
@jhrcek When I wrote the test for this I noticed the file had the SpaceAPI injected twice. It doesn't actually fix anything, it was just not used.

And this line was accidentally removed during the Project Orientated changes. So added it back.